script/agencies_verify.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import requests
+import sys
+import yaml
+
+sys.path.append("services/gtfs-rt-archive/src/gtfs_rt_archive/")
+
+from mapperfns import map_agencies_urls, map_headers  # noqa
+
+USAGE = """
+Usage:
+python script/agencies_verify.py agencies.yml headers.yml [DIFF]
+
+Arguments:
+agencies.yml - Yaml file containing agencies to download
+headers.yml - Yaml file mapping headers onto urls in agencies.yml
+DIFF - optional git diff between two hashes (will check all urls if not provided)
+"""
+
+
+def main():
+    logger = logging.getLogger("gtfs-rt-archive")
+    successes = []
+    fails = []
+    changed_urls = None
+
+    with open(sys.argv[1], "r") as f:
+        agencies = map_agencies_urls(logger, yaml.load(f, Loader=yaml.SafeLoader))
```

Comment:
This method will only result in realtime URLs being returned. We also need to check the static URLs.

Reply:
Good catch. I fixed this in a677fd3 by adding a "key_prefix" kwarg to the mapper function which defaults to the rt value (since it's part of the rt-archiver code). I verified it locally that it now catches broken schedule urls.